telegram/run_telegram_bot.js

```diff
@@ -71,8 +73,50 @@ module.exports = (args, cbk) => {
       },
 
       // Get associated LNDs
-      getLnds: ['validate', ({}, cbk) => {
-        return getLnds({logger: args.logger, nodes: args.nodes}, cbk);
+      getLnds: ['validate', async ({}) => {
+        //Use default macaroon if budget is set
+        if (!!args.payments_limit) {
+          return await getLnds({logger: args.logger, nodes: args.nodes});
+        }
+
+        const nodes = args.nodes;
+        
+        //if no saved node is specified, use the default node
+        if (!nodes || !nodes.length) {
+          const credentials =  await lndCredentials({
+            logger: args.logger,
```

Comment:
Sure. Didn't quite understand how the expiry would help since there would be multiple restarts in an year anyway?

Reply:
Yeah I would assume it would be restarted so it couldn't hurt anything, but like imagine you forgot about a telegram bot somewhere, eventually it would stop by itself